pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/ClinicalKitDao.java

```diff
@@ -29,8 +33,24 @@ public class ClinicalKitDao {
                     + "LEFT JOIN ddp_participant as p on (p.participant_id = inst.participant_id) "
                     + "LEFT JOIN ddp_instance as ddp on (ddp.ddp_instance_id = p.ddp_instance_id) "
                     + "LEFT JOIN sm_id_type sit on (sit.sm_id_type_id = sm.sm_id_type_id) "
-                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) WHERE sm.sm_id_value = ? "
-                    + "AND NOT sm.deleted <=> 1 ";
+                    + "LEFT JOIN kit_type ktype on ( sit.kit_type_id = ktype.kit_type_id) "
+                    + "WHERE sm.sm_id_value = ? AND NOT sm.deleted <=> 1 ";
+    private static final String SQL_GET_RECEIVED_CLINICAL_TISSUE_BY_DDP_PARTICIPANT_ID = "SELECT kit.received_date FROM ddp_kit "
```

Comment:
I think the select should be on sm_id instead of kit for this one

Reply:
```SELECT sm.received_date from sm_id sm LEFT JOIN ddp_tissue t on (t.tissue_id = sm.tissue_id) "```

Reply:
good catch! Thank you!!!